godu.go

```diff
@@ -27,8 +30,20 @@ func main() {
 	if err != nil {
 		log.Fatalln(err.Error())
 	}
-	log.Printf("godu will walk through `%s` that might take up to few minutes\n", rootFolderName)
-	rootFolder := core.WalkFolder(rootFolderName, ioutil.ReadDir, getIgnoredFolders())
+	progress := new(int32)
+	finished := new(int32)
+	writer := uilive.New()
+	writer.Out = os.Stderr
```

Comment:
this prevents the update output to be written on `Stdout` and being passed for example to `rm`